cms/views_test.py

```diff
@@ -36,13 +36,14 @@ def test_home_page_view(client):
     resp = client.get(page.get_url())
     content = resp.content.decode("utf-8")
 
-    assert (
-        f'<a id="actionButton" class="btn btn-primary text-uppercase px-5 py-2 action-button" href="#">Watch Now</a>'
-        in content
-    )
+    if page.about_mit_xpro and page.about_mit_xpro.video_url:
```

Comment:
I believe, we do not need that condition in test. 
Just need to make the data properly to verify the `Watch Now` button should not be in content if there is not any `video_url` and also vice versa.